assets/sass/components/dashboard/_googlesitekit-lean-cta-banner.scss

```diff
@@ -67,21 +67,13 @@
 					margin-left: $grid-gap-desktop;
 				}
 
-				.googlesitekit-audience-connect-analytics-cta__text {
+				.googlesitekit-lean-cta-banner--body__text {
```

Comment:
I've also proposed removing this class name and applying its styles to all child paragraph tags of the body, as it seems this class name is being applied to all paragraph tags anyway. Let me know what you think, thank you!